packages/commonwealth/server/routes/communities/get_related_communities_handler.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  GetRelatedCommunitiesOptions,
+  GetRelatedCommunitiesResult
+} from '../../controllers/server_communities_methods/get_related_communities';
+import { ServerControllers } from '../../routing/router';
+import { success, TypedRequestQuery, TypedResponse } from '../../types';
+
+type GetRelatedCommunitiesParams = GetRelatedCommunitiesOptions;
```

Comment:
`GetRelatedCommunitiesParams` should ideally be renamed to `GetRelatedCommunitiesQuery`, since params would refer to URL params such as `/aaa/:myParam/bbb`